pegasus/sites.v3/code.org/public/teacher-dashboard/section_assessments.haml

```diff
@@ -6,21 +6,74 @@ content-type: text/ng-template
 
 %br/
 
+
 .loading{'ng-hide' => 'assessmentsLoaded'}
   = I18n.t('dashboard_landing_loading')
 
 %div{'ng-show' => 'assessmentsLoaded'}
   = I18n.t('dashboard_assessments_view')
   %select{"ng-model" => "course_id", "ng-options"=>"course.id as course.name group by course.category for course in course_list | orderBy:['category', 'position', 'name']", 'ng-change' => 'changeScript(course_id)'}
 
-%br/
+:ruby
+  surveys_filtered = "surveyLevels | orderBy:orderSurveyAnswers | filter:(stageFilterSurveyAnswers && {stage: stageFilterSurveyAnswers}):true"
+
+%h1{'ng-show' => 'surveys.length'}= I18n.t('dashboard_surveys_answers')
+
+%div{style: "padding-left:20px", 'ng-show' => 'surveys.length'}
+  %table{style: "width:950px; table-layout:fixed;", 'ng-show' => 'surveys.length', "ng-form" => "allForm"}
+    %colgroup
+      %col{width: "250"}/
+      %col{width: "75"}/
+      %col{width: "250"}/
+      %col{width: "225"}/
+      %col{width: "75"}/
```

Comment:
Some of this cleanup might not be worth doing now if we plan to convert these views to React in the future.